src/tox/session/cmd/legacy.py

```diff
@@ -105,6 +105,7 @@ def legacy(state: State) -> int:
         return list_env(state)
     if option.devenv_path:
         option.devenv_path = Path(option.devenv_path)
+        option.env = option.env or CliEnv("py")
```

Comment:
Why this and just alter env_list?

Reply:
I wasn't able to find an easier option -- could you point me in the right direction?

Reply:
 just alter env_list via memory loader?

Reply:
I... don't know what that means